zh.js

```diff
@@ -1,747 +1,747 @@
+settings.crosshair.shadow.thickness 阴影粗细
+settings.crosshair.size 尺寸
+settings.crosshair.thickness 粗细
+settings.crosshair.gap 中心间隔大小
+settings.crosshair.dot 点
+settings.crosshair.opacity.use 使用自定义透明度
+settings.crosshair.opacity.custom 自定义透明度
+settings.hitmarker.header 命中标记
+settings.hitmarker.color 颜色
+settings.hitmarker.kcolor 击杀颜色
+settings.hitmarker.thickness 粗细
+settings.hitmarker.opacity 透明度
+settings.hitmarker.leng 长度
+settings.hitmarker.spacing 间距
+settings.hitmarker.animD 动画尺寸
+settings.hitmarker.animS 动画速度
+settings.hitmarker.hitm 显示
+settings.hitmarker.hitmFad 褪去速度
+settings.hitmarker.custom 命中标记图标
+settings.hitmarker.custom.paste 命中标记图标网址
+settings.onKillS.style.0 默认
+settings.onKillS.style.1 降低分辨率
+settings.onKillS.style.2 降低FOV
+settings.onKillS.style.3 增加FOV
+settings.serverb.header 游戏房间浏览器
+settings.serverb.oldBrowser 经典房间浏览器
+settings.serverb.defaultRegionOnly 只显示默认地区的房间
+settings.gameprivacy.header 隐私
+settings.gameprivacy.streamer 主播模式
+settings.gameprivacy.anon 匿名模式
+settings.gameprivacy.hideGameTitle 隐藏游戏标志
+settings.gameplay.header 游戏性
+settings.gameplay.sensitivityX X轴灵敏度
+settings.gameplay.sensitivityY Y轴灵敏度
+settings.gameplay.aimsensX ADS(瞄准)X轴灵敏度
+settings.gameplay.aimsensY ADS(瞄准)Y轴灵敏度
+settings.gameplay.challenge 挑战模式
+settings.gameplay.invert 反转Y轴
+settings.gameplay.onkillS 击杀
+settings.gameplay.scrollDir 鼠标滚轮方向
+settings.controller.header 手柄
+settings.controller.sensitivityX X轴灵敏度
+settings.controller.disableCons 禁用手柄
+settings.controller.sensitivityY Y轴灵敏度
+settings.controller.aimsensX ADS(瞄准)X轴灵敏度
+settings.controller.aimsensY ADS(瞄准)Y轴灵敏度
+settings.controller.deadZoneL 手柄摇杆死区 (左)
+settings.controller.deadZoneR 手柄摇杆死区 (右)
+settings.controller.triggerThres 触发阈值
+settings.controller.vibration 震动
+settings.controller.gradSpeed 渐缓速度
+settings.controller.invert 反转Y轴
+settings.mobile.header 移动设备
+settings.mobile.sensitivityX X轴灵敏度
+settings.mobile.sensitivityY Y轴灵敏度
+settings.mobile.aimsensX ADS(瞄准)X轴灵敏度
+settings.mobile.aimsensY ADS(瞄准)Y轴灵敏度
+settings.mobile.vibration 震动
+settings.mobile.gradSpeed 渐缓速度
+settings.mobile.invert 反转Y轴
+settings.audio.header 音效
+settings.audio.sound 主音量
+settings.audio.ambient 背景音量
+settings.audio.voice 语言音量
+settings.audio.weps 武器音量
+settings.audio.player 玩家音量
+settings.audio.skin 玩家服饰音量
+settings.audio.ui 用户界面音量
+settings.audio.asset 地图音量
+settings.audio.dialogue 动作音量
+settings.viewmodel.header 视角
+settings.viewmodel.fov 腰射视野
+settings.viewmodel.fps 帧数
+settings.viewmodel.bobbing 玩家移动时的武器晃动
+settings.viewmodel.leaning 玩家移动时的武器倾斜
+settings.viewmodel.show.primary 显示主武器
+settings.viewmodel.show.secondary 显示副武器
+settings.viewmodel.roundArms 圆形手臂
+settings.viewmodel.show.melee 显示近战武器
+settings.viewmodel.show.ads 开镜时隐藏武器
+settings.viewmodel.toggleAim 松手时保持开镜
+settings.viewmodel.show.hands 显示手部
+settings.viewmodel.leftHand 左手持枪
+settings.viewmodel.aimAnim 武器开镜动画
+settings.viewmodel.rot 武器位置旋转
+settings.viewmodel.xOff 武器X轴坐标 (左/右）
+settings.viewmodel.yOff 武器Y轴坐标（上/下）

```

Comment:
`(` and `（` are different, are you able to fix this up please?

Ideally go with `(` and `)` to help reduce whitespace

Reply:
can you fix this one through out the whole file? find and replace should do the trick!